astropy/cosmology/_src/traits/darkenergy.py

```diff
@@ -141,17 +138,15 @@ def Ode(self, z: Quantity | ArrayLike) -> FArray | float:
 
         Returns
         -------
-        Ode : ndarray or float
+        Ode : ndarray
             The density of dark energy relative to the critical density at each
             redshift.
-            Returns `float` if the input is scalar.
         """
         z = aszarr(z)
         if self.Ode0 == 0:  # Common enough to be worth checking explicitly
-            return np.zeros(z.shape) if hasattr(z, "shape") else 0.0
+            return np.zeros(z.shape)
```

Comment:
So maybe again `np.zeros_like(z)`. Though again I fail to see why you want to preserve the shape at all.

Reply:
Done, thanks!